app/services/hyku_addons/jwt_cookies_service.rb

```diff
@@ -14,8 +14,9 @@ def initialize(user, env)
       @request = env.request
     end
 
-    def set_jwt_cookies(user = nil)
+    def assign_jwt_cookies(user = nil)
```

Comment:
This renaming does clear up the code.  Did rubocop spot this because it doesn't like setters named with set_? If that's so how did the private method get passed?